### DISCUSSION TITLE
Welcome to Orkestra Discussions!

### DISCUSSION THREAD

**Discussion by jwillp** [Morebec/OrkestraMono#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by silviuvoicu** [comment]
Hi,
Do you have an example of how to use Orkestra?
Maybe present a just little case of some sort of problem, that using Orkestra may help to solve :)

**Reply by jwillp** [reply]
Hi @silviuvoicu, thank you for your interest in Orkestra! 
Not at the moment, but we are currently working on creating a demo app with a tutorial
showcasing how you can create a DDD/CQRS/ES application with Orkestra and Symfony.

We are looking for a more interesting example than a classic Todo List, but it might be the better option than nothing for the time being.
